code/LINDA/LINDA_turf_tile.dm

```diff
@@ -305,17 +305,24 @@
 
 
 /atom/movable/var/pressure_resistance = 5
+/atom/movable/var/throw_pressure_limit = 15
 /atom/movable/var/last_forced_movement = 0
 
 /atom/movable/proc/experience_pressure_difference(pressure_difference, direction)
 	if(last_forced_movement >= air_master.current_cycle)
 		return 0
 	else if(!anchored && !pulledby)
-		if(pressure_difference > pressure_resistance)
+		if(pressure_difference >= throw_pressure_limit)
+			var/general_direction = get_edge_target_turf(src,direction)
+			if(ismob(src)) to_chat(src, "<span class='userdanger'>The pressure sends you flying!</span>")
```

Comment:
@Allfd the opposite, actually, you were using `spawn` in-line, he wanted it to be ```
spawn
   throw_at()
```